src/MediaGenerator/Command/AbstractCitizenProjectMediaCommand.php

```diff
@@ -24,7 +24,7 @@
     /**
      * @var File
      *
-     * @Assert\Image(minWidth=1200)
+     * @Assert\Image(minWidth=1600)
```

Comment:
I think not need change this, it's for generate an image for facebook cover which should have 1200px for width